### PR TITLE
implemented creating an employee and also creating a company together…

### PR DESCRIPTION
**Pull request by Ayomipo18**
… with employees using post request

### CODE REVIEW

File: CompanyEmployees.Presentation/Controllers/EmployeesController.cs [hunk 1]
```diff
@@ -22,11 +23,22 @@ public IActionResult GetEmployeesForCompany(Guid companyId)
             return Ok(employees);
         }
 
-        [HttpGet("{id:guid}")]
+        [HttpGet("{id:guid}", Name = "GetEmployeeForCompany")]
```
Comment (anabiz) [inline CompanyEmployees.Presentation/Controllers/EmployeesController.cs:26]: You can just name this method "Get employee"

File: CompanyEmployees.Presentation/Controllers/EmployeesController.cs [hunk 2]
```diff
@@ -22,11 +23,22 @@ public IActionResult GetEmployeesForCompany(Guid companyId)
...
         {
             var employee = _service.EmployeeService.GetEmployee(companyId, id, trackChanges: false);
             return Ok(employee);
         }
+
+        [HttpPost]
+        public IActionResult CreateEmployeeForCompany(Guid companyId, [FromBody] EmployeeForCreationDto employee)
+        {
+            if (employee is null)
+                return BadRequest("EmployeeForCreationDto object is null");
```
Comment (anabiz) [inline CompanyEmployees.Presentation/Controllers/EmployeesController.cs:37]: This DTO can simply be named "EmployeeCreateDto"

**Review by anabiz — CHANGES_REQUESTED** [review]

File: CompanyEmployees.Presentation/Controllers/EmployeesController.cs (hunk 1 above)
Comment (Ayomipo18) [reply CompanyEmployees.Presentation/Controllers/EmployeesController.cs]: i have renamed GetEmployeeForCompany to GetEmployee

File: CompanyEmployees.Presentation/Controllers/EmployeesController.cs (hunk 2 above)
Comment (Ayomipo18) [reply CompanyEmployees.Presentation/Controllers/EmployeesController.cs]: I have renamed both Employee and Company Dtos to EmployeeCreateDto and CompanyCreateDto respectively.

**Review by anabiz — APPROVED** [review]

**Pull request merged**